### PR TITLE
Adds whitelist README.

### LINKED ISSUE #161: Add a whitelist README
:Add note to contributors of whitelists about what to do. Here is the basic guidance:

* Use the standard [fork and pull](https://reflectoring.io/github-fork-and-pull/) approach.
* Make a list of all phones or phonemes, in descending-frequency order, using the appropriate file in `data/wikipron/tsv`.
* Remove any rare, bizarre, and non-native sounds.
* For phonemic whitelists, annotate reasonably-frequent allophones with a comment. This is optional for phonetic whitelists.
* Run `./postprocess && ./generate_summary.py`.
* `git add` the whitelist, the newly-filtered TSV files, and the README and summary TSVs modified, then commit, push, and file the PR. This will also allow the reviewer to quickly see how many forms are removed by whitelisting.

Comment style is Python-like: phone/phoneme, two spaces, `#`, space, then a title-cased expression with period at end. E.g., `t̪  # Allophone of /t/.`

### PR DESCRIPTION
**Pull request by kylebgorman**
Closes #161.

- [ ] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

### CODE REVIEW

File: data/wikipron/whitelist/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+up poorly with the language-specific Wiktionary transcription guidelines.
+
+How to submit a whitelist
+-------------------------
+
+We welcome user submissions for whitelists from linguists. Note that we use the
+[fork and pull](../../../CONTRIBUTING.md) model for contributions.
+
+1.  Make a list of all phones or phonemes, in descending-frequency order, using
+    the appropriate file in [`../tsv`](../tsv).
```
Comment (jacksonllee) [inline data/wikipron/whitelist/README.md:38]: Is the contributor on their own to, say, generate this said list of phones/phonemes in descending frequency order based on a TSV data file not yet processed by this whitelist step? Or do we already have tools to facilitate this?

File: data/wikipron/whitelist/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    `git add`.
+6.  Commit using `git commit`, push to your branch using `git push`, and then
+    file a pull request.
+
+The whitelist format is a UTF-8 encoded file with one segment per line, with
+optional comments formatted as two spaces, `#`, one space, and then a sentence
+or sentence fragment with appropriate punctuation (e.g.,
+`tʰ  # Allophone of /t/.`). Please do not leave any trailing whitespace. The
+whitelist file should have the same name as the corresponding TSV file, but with
+a `.whitelist` extension instead of `.tsv`.
```
Comment (jacksonllee) [inline data/wikipron/whitelist/README.md:52]: Potential small engineering work if a student is interested: some (all?) of these assumptions for the whitelist files could be formally enforced as some kind of whitelist validation run by the test suite. Maybe it's worth a good-first-feature-and-help-wanted sort of ticket, but maybe it's not, no strong preference from me.

File: data/wikipron/whitelist/README.md (hunk 1 above)
Comment (kylebgorman) [reply data/wikipron/whitelist/README.md:38]: Not yet but that's an obvious future improvement.

File: data/wikipron/whitelist/README.md (hunk 2 above)
Comment (kylebgorman) [reply data/wikipron/whitelist/README.md:52]: Good idea, yeah. I'll try to get someone interested.

**Pull request merged**